doc/source/admin/dependency_resolvers.rst

```diff
@@ -0,0 +1,143 @@
+Dependency Resolvers in Galaxy
+==============================
+
+There are two parts to building a link between Galaxy and command line bioinformatics tools: the tool XML that
+specifies a mapping between the Galaxy web user interface and the tool command line and tool dependencies that specify
+how to source the actual packages that implement the tool’s commands. The final script that Galaxy submits to run a
+the job uses includes commands, such as changes to the ``PATH`` environment variable, that are generated by *dependency
```

Comment:
Remove `the` at the start of the line (there's already an `a` at the end of the previous line) and `uses`.